openedx/core/lib/api/permissions.py

```diff
@@ -56,5 +56,18 @@ def has_permission(self, request, view):
         # other users, do not let them deduce the existence of an account.
```

Comment:
Update documentation so it is clear that a 404 is raised for users with is_staff == True.